docs/public-networks/how-to/use-configuration-file/profile.md

```diff
@@ -15,55 +30,44 @@ besu --config-file=config.toml --profile=staker
-the same options set in the profile.
-See the [configuration order of precedence](index.md#configuration-order-of-precedence) for more information.
-
-You can use the following profiles:
-
-- [Minimalist staker profile](#minimalist-staker-profile)
-- [Staker profile](#staker-profile)
-- [Enterprise/Private profile](#enterpriseprivate-profile)
+ -->
 
 ## Minimalist staker profile
 
-For stakers who want to maximize their hardware value but don't want to serve full sets of data to
-their peers, Besu provides a minimalist staker profile.
-See the
+`--profile=minimalist_staker` is optimized for stakers who want to maximize their hardware value but don't want to serve full sets of data to their peers, See the
 [minimalist staker profile on GitHub](https://github.com/hyperledger/besu/blob/8b64023a121ea996ef60e4b7e2299c5807683f90/config/src/main/resources/profiles/minimalist-staker.toml)
 for the custom settings.
 
+<!-- IMO this is all redundancy -- fully supported by the linked page above
+
 To use the minimalist staker profile, run Besu with
 [`--profile`](../../reference/cli/options.md#profile) set to `minimalist_staker`:
 
 ```bash
 besu --profile=minimalist_staker
-```
+``` -->
 
 ## Staker profile
 
-For stakers who want to maximize their hardware value and also want to serve full sets of data to
-their peers, Besu providers a staker profile.
-See the
+`--profile=staker` is optimized for stakers who want to maximize their hardware value while also serving full sets of data to their peers. See the
 [staker profile on GitHub](https://github.com/hyperledger/besu/blob/8b64023a121ea996ef60e4b7e2299c5807683f90/config/src/main/resources/profiles/staker.toml)
 for the custom settings.
 
+<!-- IMO this is all redundancy -- fully supported by the linked page above
+
 To use the staker profile, run Besu with [`--profile`](../../reference/cli/options.md#profile) set to `staker`:
 
 ```bash
 besu --profile=staker
-```
+``` -->
 
 ## Enterprise/Private profile
 
-For private network operators who want to minimize confusion by setting sensible defaults, Besu
-provides an enterprise/private profile.
-This profile is designed to handle specific use cases for private network operators.
-See the
-[enterprise/private profile on GitHub](https://github.com/hyperledger/besu/blob/8b64023a121ea996ef60e4b7e2299c5807683f90/config/src/main/resources/profiles/enterprise-private.toml)
+`--profile=private` / `--profile=enterprise` supports private network operators and enterprises by handling specific use cases that apply to private network operators. See the [enterprise/private profile on GitHub](https://github.com/hyperledger/besu/blob/8b64023a121ea996ef60e4b7e2299c5807683f90/config/src/main/resources/profiles/enterprise-private.toml)
 for the custom settings.
 
+<!-- IMO this is all redundancy -- fully supported by the linked page above
+
 To use the enterprise/private profile, run Besu with
```

Comment:
I think this link should be included and not commented out.

Reply:
OK -- going to keep single source but duplicate links to that source